### DISCUSSION TITLE
My Computer Bootloops On Loading Brunch Framework Screen

### DISCUSSION THREAD

**Discussion by CrossyAtom46** [sebanc/brunch#1453] [Q&A]
When i came to Loading Brunch Framework Computer reboots then retrying to load brunch framework then Loading Brunch Framework then reboots every time happening this i tried chromeos_14268.67.0_grunt_recovery_stable-channel_mp-v6.bin file for  Chrome OS

My PC Specs:

Specs:

Display: 13.3", 1366 x 768 LED backlit display. AMD Radeon HD8250 graphics (part of the AMD APU package, similar to integrated graphics in terms of technology). Micro HDMI and VGA port via optional adapter.

Battery: 2 cell, 30 Whr, 5880 mAh Lithium Ion rechargeable, sealed inside.

Performance: 1 GHz AMD A6 quad core APU with boost to 1.4 GHz. 4 gigs 1066MHz DDR3L RAM and 128 gig SATA3 SSD drive.

Size: 12.76 x 8.82 x 0.67 inches. Weight: 3.17 pounds.

What Should I Do?

**Comment by tejasraman** [comment]
Trying to run Brunch on an A6-1540 laptop APU(or similar OEM variant) is like me running Brunch on an A10-6800K.
It is **incompatible**
